sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2072,6 +2072,11 @@ object SQLConf {
       .stringConf
       .createWithDefault(
         "https://maven-central.storage-download.googleapis.com/repos/central/data/")
+
+  val SPARK_SQL_CLI_SHOW_CURRENT_DB = buildConf("spark.sql.cli.show.current.db")
```

Comment:
`SPARK_SQL_CLI_SHOW_CURRENT_DB_ENABLED` and `spark.sql.cli.show.current.db.enabled`?